deploy/runtools/runtime_config.py

```diff
@@ -287,6 +401,15 @@ def __init__(self, runtimeconfigfile: str, configoverridedata: str) -> None:
             rootLogger.warning(overridefield + "=" + overridevalue)
             runtime_dict[overridesection][overridefield] = overridevalue
 
+        self.metasimulation_enabled = False
+        self.metasimulation_host_simulator = 'verilator'
+        if 'metasimulation' in runtime_dict:
+            metasim_dict = runtime_dict['metasimulation']
+            if 'metasimulation_enabled' in metasim_dict:
+                self.metasimulation_enabled = metasim_dict['metasimulation_enabled']
+            if 'metasimulation_host_simulator' in metasim_dict:
+                self.metasimulation_host_simulator = metasim_dict['metasimulation_host_simulator']
```

Comment:
IMO I would just enforce that the `metasim` section and all its subfields exist. Then you don't need to have defaults in the python

Reply:
fixed